uportal-war/src/main/resources/layout/theme/respondr/content.xsl

```diff
@@ -322,12 +322,34 @@
    | These two templates render the focused portlet content.
   -->
   <xsl:template match="focused">
+    <!--
+        Check for the existance of the "focused" portlet in a region.  If one is found, prevent the portlet from rendering
+        in the content portion of the layout.  Test for a match against the fname, ID, and channel ID.
+    -->
+    <xsl:variable name="focusedFname">
+        <xsl:value-of select="//focused/channel/@fname"/>
+    </xsl:variable>  
+
+    <xsl:variable name="focusedChannelID">
+        <xsl:value-of select="//focused/channel/@chanID"/>
+    </xsl:variable>
+
+    <xsl:variable name="focusedID">
+        <xsl:value-of select="//focused/channel/@ID"/>
+    </xsl:variable>
+
+    <xsl:variable name="focusedMatchTest">
+        <xsl:value-of select="not(((//region/channel[@fname=$focusedFname]) and (//region/channel/@chanID=$focusedChannelID)) and (//region/channel[@ID=$focusedID]))"/>
```

Comment:
Does this have to be this complicated?  I admit I'm not fully understanding what this code is trying to do, but stepping back a bit -- either matching on portlet publication matters, so that would just be `@fname`, or matching on window ID matters, so that's, what, `@ID`?